lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -0,0 +1,237 @@
+        except Exception as e:
+            self.manageiq.module.fail_json(msg="Failed to delete user {userid}: {error}".format(userid=userid, error=e))
+
+    def user_update_required(self, user, username, group_id, email):
+        """ Returns True if the username, group id or email passed for the user
+            differ from the user's existing ones, False otherwise.
+        """
+        if username is not None and user['name'] != username:
+            return True
+        if group_id is not None and user['current_group_id'] != group_id:
+            return True
+        if email is not None and user.get('email') != email:
+            return True
+        return False
+
+    def update_user_if_required(self, user, userid, username, group_id, password, email):
+        """Updates the user in manageiq.
+
+        Returns:
+            the created user id, name, created_on timestamp,
+            updated_on timestamp, userid and current_group_id
+        """
+        if not self.user_update_required(user, username, group_id, email):
+            return dict(
+                changed=False,
+                msg="User {userid} already exist, no need for updates".format(userid=userid))
+        url = '{api_url}/users/{user_id}'.format(api_url=self.manageiq.api_url, user_id=user.id)
+        resource = {'userid': userid, 'group': {'id': group_id}}
+        if username is not None:
+            resource['name'] = username
+        if password is not None:
+            resource['password'] = password
+        if email is not None:
+            resource['email'] = email
+
+        try:
+            result = self.manageiq.client.post(url, action='edit', resource=resource)
+        except Exception as e:
+            self.manageiq.module.fail_json(msg="Failed to update user {userid}: {error}".format(userid=userid, error=e))
+        return dict(
+            changed=True,
+            msg="Successfully updated the user {userid}: {user_details}".format(userid=userid, user_details=result))
+
+    def create_user(self, userid, username, group_id, password, email):
+        """Creates the user in manageiq.
+
+        Returns:
+            the created user id, name, created_on timestamp,
+            updated_on timestamp, userid and current_group_id
+        """
+        url = '{api_url}/users'.format(api_url=self.manageiq.api_url)
+        resource = {'userid': userid, 'name': username, 'password': password,
+                    'group': {'id': group_id}, 'email': email}
+        try:
+            result = self.manageiq.client.post(url, action='create', resource=resource)
+        except Exception as e:
+            self.manageiq.module.fail_json(msg="Failed to create user {userid}: {error}".format(userid=userid, error=e))
+        return dict(
+            changed=True,
+            msg="Successfully created the user {userid}: {user_details}".format(userid=userid, user_details=result['results']))
```

Comment:
results are the return value of the `manageiq.client.post`, if the api call returns the new user object, it will returned as `user_details`

when running the example I get:
```
changed: [127.0.0.1] => {
    "changed": true, 
    "failed": false, 
    "invocation": {
        "module_args": {
            "action": "create", 
            "email": "jdoe@example.com", 
            "group": "EvmGroup-user", 
            "miq": {
                "password": "smartvm", 
                "url": "http://127.0.0.1:3000", 
                "username": "admin", 
                "verify_ssl": false
            }, 
            "name": "Jane Doe", 
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER", 
            "userid": "jdoe"
        }
    }, 
    "msg": "Successfully created the user jdoe: [{u'name': u'Jane Doe', u'settings': {}, u'userid': u'jdoe', u'current_group_id': u'4', u'id': u'6', u'created_on': u'2017-07-26T09:54:45Z', u'href': u'http://127.0.0.1:3000/api/users/6', u'updated_on': u'2017-07-26T09:54:45Z', u'email': u'jdoe@example.com'}]"
}

```